dev/ColorPicker/ColorPicker.xaml

```diff
@@ -10,122 +10,13 @@
     <Style TargetType="controls:ColorPicker" BasedOn="{StaticResource DefaultColorPickerStyle}"/>
     
     <Style x:Key="DefaultColorPickerStyle" TargetType="controls:ColorPicker">
-        <Setter Property="MaxWidth" Value="392" />
-        <Setter Property="MinWidth" Value="312" />
         <Setter Property="IsTabStop" Value="False" />
         <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="controls:ColorPicker">
-                    <Grid Background="{TemplateBinding Background}">
+                    <Grid x:Name="RootGrid" Background="{TemplateBinding Background}">
                         <Grid.Resources>
-                            <Style x:Key="ColorPickerSliderStyle" TargetType="primitives:ColorPickerSlider">
```

Comment:
@StephenLPeters @karenbtlai do you know if there was a reason for having a custom slider style here ?

Reply:
It's for the slider to have the gradient colours rather than just a solid bar with the accent colour.